### DISCUSSION TITLE
Can I run bloop within a docker container and accessed it from the host?

### DISCUSSION THREAD

**Discussion by attilapiros** [scalacenter/bloop#1822] [Q&A]
Use case: assume you have a machine with M1 processor but you have to use a component where JNI is used but in that component there is no support for 64bit architecture. 
One of the example is LevelDB  (with [leveldbjni](https://github.com/fusesource/leveldbjni)). 
This way Rosetta does not really help but a docker container does the trick.

Is it enough to map the bloop port in such a setup or does it need access to some files (from the host where my project files is really stored)?

**Comment by tgodzik** [comment]
Bloop needs to access the file that it needs to compile, we assume that it's available in the current filesystem. It should be possible to run Bloop in the container and map the port to the local one. I think that should do the trick.
